game_event_player/tests.py

```diff
@@ -0,0 +1,168 @@
+        username='testuser3',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+def saved_player(user):
+    player = Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+    return player
+
+
+@pytest.fixture
+def saved_player2(user2):
+    player = Player.objects.create(
+        user=user2,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+    return player
+
+
+@pytest.fixture
+def saved_player3(user3):
+    player = Player.objects.create(
+        user=user3,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+    return player
+
+
+@pytest.fixture
+def players(saved_player, saved_player2, saved_player3):
+    return tuple([saved_player, saved_player2, saved_player3])
+
+
+@pytest.fixture
+def saved_court():
+    court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
+                                 city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+    return court
+
+
+@pytest.fixture
+def saved_game_event(saved_court):
+    game_event = GameEvent.objects.create(id=ID1, time=datetime.today(), level_of_game=3,
+                                          min_number_of_players=2, max_number_of_players=5, court_id=saved_court,
+                                          ball_game='Basketball')
+    return game_event
+
+
+@pytest.fixture
+def saved_game_event2(saved_court):
+    game_event = GameEvent.objects.create(id=ID2, time=datetime.today(), level_of_game=3,
+                                          min_number_of_players=2, max_number_of_players=5, court_id=saved_court,
+                                          ball_game='Basketball')
+    return game_event
+
+
+@pytest.fixture
+def saved_game_event_player(saved_game_event, saved_player):
+    return GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player,
+                                          ball_responsible=False)
+
+
+@pytest.fixture
+def create_game_event_players(saved_game_event, saved_players):
+    players = [GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=player)
+               for player in saved_players]
+    return players
+
+
+@pytest.mark.django_db
+class TestGameEventPlayerModel:
+
+    def test_game_event_player_ball_responsible_default_value(self, saved_game_event, saved_player):
+        GameEventPlayer.objects.create(game_event_id=saved_game_event, player_id=saved_player, ).full_clean()
+
+    def test_game_event_player_creation(self, saved_game_event, saved_player):
+        game_event_player = GameEventPlayer.objects.create(game_event_id=saved_game_event,
+                                                           player_id=saved_player,)
+        assert game_event_player.game_event_id == saved_game_event
+        assert game_event_player.player_id == saved_player
+        assert game_event_player.ball_responsible is False
```

Comment:
You can make this change and then you can delete the test case above. Its name doesn't make sense and this line will also make it redundant.
```suggestion
    def test_game_event_player_creation(self, saved_game_event, saved_player):
        game_event_player = GameEventPlayer.objects.create(game_event_id=saved_game_event,
                                                           player_id=saved_player,)
        game_event_player.full_clean()
        assert game_event_player.game_event_id == saved_game_event
        assert game_event_player.player_id == saved_player
        assert game_event_player.ball_responsible is False
```